### PR TITLE
Bump used node version and builder image

### PR DESCRIPTION
**Pull request by abeforgit**

### CODE REVIEW

**Comment by abeforgit** [comment]
Fun fact: `npm ci` also runs `prepare` since npm 8

File: Dockerfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ LABEL maintainer="info@redpencil.io"
 
 WORKDIR /app
 COPY package.json package-lock.json ./
+COPY public ./public/
```
Comment (nvdk) [inline Dockerfile:7]: why was this necessary?

Comment (abeforgit) [reply Dockerfile:7]: because new npm now runs prepare scripts on `npm ci` :) which in our case includes the `svg-symbols` invocation, which needs the public folder to exist
